### ISSUE TITLE
twilight menu ++ doesn't open nds roms on first try on my dsi

### ISSUE THREAD

**Issue by Qwertymom** [DS-Homebrew/TWiLightMenu#933]
**Your console configuration:**  DSI
**TWiLight Menu++ Version:** release v0,212
**TWiLight Menu++ Theme:** 3ds

### Expected behaviour

The DSI should run nds roms as expected

### Actual behaviour

ever since I downloaded twilight menu++ on my dsi, any and all nds roms I run won't work, it just shows the loading screen and then a white screen. It will work when I soft reset my dsi, then bootup twilight++ via leapnote and then run that EXACT same rom again from there. If I run any other nds rom from here, it will result in the same problem and I'd have to re-open twilight menu and rerun the exact same rom. This is really inconvenient because it takes a long time to boot twilight menu and makes the whole thing inconvenient so if someone has a fix to make the nds roms run on first try that'd be great. Also this might not be AS big a deal if twilight menu actually auto started on boot but I can't get that to work either for some reason... if I could get a fix for that as well it'd be great. thanks.

### Steps to reproduce

1.
2.
3.

**Comment by RocketRobz** [comment]
That issue never happens for me.

What happens when you install Unlaunch?

**Comment by Qwertymom** [comment]
I believe i have already installed unlaunch on my dsi, it's how I got twilight menu in the first place. It was quite a long time ago so I couldn't remember the specifics but it seems to be what was on the tutorial video I followed when I did it in the first place. Here's the link to the video if you're interested, maybe they set it up wrong or something?? https://www.youtube.com/watch?v=CWg0ziiueNE

**Comment by RocketRobz** [comment]
Don't follow video guides.
Follow [these](https://github.com/DS-Homebrew/TWiLightMenu/wiki/How-to-install-TWiLight-Menu) steps instead.

**Comment by Qwertymom** [comment]
Wait so are you suggesting that I clear the cfw and reinstall it?? Also how would I run twilight menu from unlaunch??

**Comment by NightYoshi370** [comment]
Yes, he is suggesting that you undo whatever you've done with that video and follow the community-supported guide.

Hold A+B, set your boot button combination to what you'd like TWiLight to be.
We could assist with support better on the discord server found on the ReadME. Please join there. We wouldn't like to be spamming everyone's emails with simple help questions

Closed

**Comment by Qwertymom** [comment]
Ok so I checked out the guide you sent, downloaded required files and deleted any folders already on my sd card that I was about to replace to ensure that they're replaced perfectly. The first game I ran seemed to work fine but then I started to run into the same problem afterwards. I took note of this and re-replaced the folders with the original, then loaded twilight menu, and turned off update recently played list (which is what I suspect did the trick) and also turned off any potentially buggy setting, and it seems to be working for the time being. A new problem has arisen now though, where running any snes or gba games results in the nds_boostrap not found error 1 problem.
